metaflow/plugins/kfp/kfp.py

```diff
@@ -239,20 +240,23 @@ def _get_retries(node: DAGNode) -> Tuple[int, int]:
         return max_user_code_retries, max_user_code_retries + max_error_retries
 
     @staticmethod
-    def _get_resource_requirements(node: DAGNode) -> Dict[str, str]:
+    def _get_pod_customization(node: DAGNode) -> Dict[str, str]:
         """
         Get resource request or limit for a Metaflow step (node) set by @resources decorator.
+        Get pod labels and annotations from @pod_label and @pod_annotation decorators
 
+        For resources:
         Supported parameters: 'cpu', 'cpu_limit', 'gpu', 'gpu_vendor', 'memory', 'memory_limit'
         Keys with no suffix set resource request (minimum);
         keys with 'limit' suffix sets resource limit (maximum).
-
         Eventually resource request and limits link back to kubernetes, see
         https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
-
         Default unit for memory is megabyte, aligning with existing resource decorator usage.
 
-        Example using resource decorator:
+        Example usage handled by this function:
+            @pod_label(name="namespace", value="aip_kfp_example")
+            @pod_label(name="team", value="aip")
```

Comment:
name -> key

Reply:
The pod labels and annotation attributes belong in this decorator, then it's no longer `key, value` but `@kfp(labels=[], annotations=[])`

Reply:
@talebzeghmi It will still be key - value pairs. For example:
```
@kfp(
    labels={"team": "aip", "type": "training"},
    annotation={"description": "Example flow for annotation and labels"}
)
```

Reply:
Oh I see what you mean, we are directly putting a dictionary here instead of using the var `name` or `key`.